extensions/wikia/Recirculation/styles/mixed-content-footer.scss

```diff
@@ -135,6 +137,43 @@ h1.mcf-header {
 	}
 }
 
+.mcf-en .mcf-card {
+	&[data-tracking*="card-6"],
```

Comment:
please do not use data-tracking to style elements
add separate class to elements